LICENSE.txt

```diff
@@ -1,5 +1,5 @@
 
-    Copyright 2013-2022 the original author or authors from the JHipster project
+    Copyright 2018-2021 the original author or authors from the JHipster project
```

Comment:
plz revert the change on LICENSE

Reply:
Oh yes sorry, i thought that was a typo or a copy/paste error, i will revert